src/overlays/actors/ovl_En_Encount1/z_en_encount1.h

```diff
@@ -3,15 +3,22 @@
 
 #include "global.h"
 
+#define ENENCOUNT1_MAX_SPAWNS_TOTAL_INFINITE 63 // aka 0x3F, All bits set
+
 #define ENENCOUNT1_GET_TYPE(thisx) (((thisx)->params >> 11) & 0x1F)
-#define ENENCOUNT1_GET_7C0(thisx) (((thisx)->params >> 6) & 0x1F)
-#define ENENCOUNT1_GET_PATH(thisx) ((thisx)->params & 0x3F)
+#define ENENCOUNT1_GET_MAX_SPAWNS_CUR(thisx) (((thisx)->params >> 6) & 0x1F)
+#define ENENCOUNT1_GET_MAX_SPAWNS_TOTAL(thisx) ((thisx)->params & 0x3F)
+#define ENENCOUNT1_GET_PATH_INDEX(thisx) ((thisx)->params & 0x3F) // Used only by EN_ENCOUNT1_SKULLFISH_2 which doesn't use MaxSpawnsTotal
+
+#define ENENCOUNT1_GET_MIN_SPAWN_TIME(thisx) ((thisx)->world.rot.x) // Time to wait between spawning
+#define ENENCOUNT1_GET_SPAWN_UNK(thisx) ((thisx)->world.rot.y) // Unused
+#define ENENCOUNT1_GET_MAX_SPAWN_DISTANCE(thisx) ((thisx)->world.rot.z) // Negative means infinite distance
```

Comment:
Of course we should; the word `GET` just designates that some data is being retrieved and isn't intrinsic to z64 params access. The behavior you see in TimeTag seems non-standard; the actor shouldn't be using the rotation variables to store persistent non-rotation data, it should be extracted in the ActorInit function like it's done within this actor.

Reply:
I agree and don't think `TimeTag` is indicative for invisible actors.  `TimeTag` seems unique since it uses the rots as struct members, instead of like params as is usual so it makes to have a common macro there, while just a getter here.